Objects/genericaliasobject.c

```diff
@@ -225,6 +226,33 @@ tuple_add(PyObject *self, Py_ssize_t len, PyObject *item)
     return 0;
 }
 
+// Makes a shallow copy of a tuple
+static inline PyObject *
+tuple_copy(PyObject *obj) {
+    return _PyTuple_FromArray(((PyTupleObject *)obj)->ob_item, Py_SIZE(obj));
```

Comment:
PEP 7 states that the `{` for function definitions should go on its own line (https://www.python.org/dev/peps/pep-0007/#code-lay-out).